packages/ripple-nuxt-tide/lib/core/tide-helper.js

```diff
@@ -100,7 +101,8 @@ export const getPageHeadConfig = ({
   image,
   imageAlt,
   siteSectionName = '',
-  pageType
+  pageType,
+  robotsMeta
```

Comment:
Hey @MdNadimHossain Do you think we can pass all meta tags from page data as a property instead of specific one? So we don't need to add more properties in the future. This function was created by me in a quick code split task didn't consider too much on the function its self. And it's only used by packages/ripple-nuxt-tide/lib/middleware/page-head.js. Feel free to improve it.

Reply:
Hey @tim-yao Correct me if I am wrong, but if we do that then we have to do the validation of them in the function itself. Now the validation is done for each of them in here `packages/ripple-nuxt-tide/lib/middleware/page-head.js` isn't it!

Reply:
@MdNadimHossain yes, you are right. It's ok to keep it as it is then. No need to over engineering it.